indico/modules/events/contributions/templates/forms/contribution_person_widget.html

```diff
@@ -82,8 +80,8 @@
 
             function renderPerson(person) {
                 var $personRow = $('<div>').addClass('person-row');
-                var $personName = $('<span>').text(person.name);
-                var $personStatus = $('<span>').addClass('person-status');
+                var $personName = $('<div>').addClass('name').text(person.name);
+                var $personStatus = $('<div>').addClass('person-status');
                 var $personRoles = $('<span>').addClass('person-roles');
                 var $personButtons = $('<span>').addClass('person-buttons');
                 var $buttonRemove = $('<a>').addClass('i-button-icon danger icon-close').attr('title', $T.gettext("Remove person"));
```

Comment:
This again? I don't have a very strong opinion, but I keep insisting that: 

1. Unless we write one-liners dicts (ugly) the instantiation of an element will take a minimum of 3 lines.
2. Even in the jQuery docs is stated that methods are a most robust interface to use.

> While the second argument is convenient, its flexibility can lead to unintended consequences
> [Source](http://api.jquery.com/jquery/#jQuery2)